### ISSUE TITLE
Keyboard shortcuts 

### ISSUE THREAD

**Issue by JonasHersom** [logicguy1/FOSS-Lectio-improvements#7]
Would be cool if you could make custom keyboard shortcuts for different pages, like "1" for "forside" and "2" for "opgaver" etc 

Also you really need to add a system to customize the colors because the current ones hurt my soul

**Comment by logicguy1** [comment]
Agreed, #5 is working on implementing a settings system then it will be included as a module of that
